packages/stripe-integration/src/stripev3/stripev3.mock.ts

```diff
@@ -276,3 +278,24 @@ export function getStripeShippingAddressGuestUserWithoutAddress(): StripeConfirm
         },
     };
 }
+
+export function getBillingAddress(): BillingAddress {
```

Comment:
You can get this mock from payment-integration-test-utils package

Reply:
done